airflow/utils/db.py

```diff
@@ -477,6 +477,19 @@ def create_default_connections(session: Session = NEW_SESSION):
         ),
         session,
     )
+    merge_conn(
+        Connection(
+            conn_id="adb_spark_default",
+            conn_type="adb_spark",
```

Comment:
Where is `abd` coming from?
I didn't find `abd` to officially used in Alibaba cloud docs.

On another front, cc also @ephraimbuddy @pierrejeambrun would you prefer to have this change in a separated PR? This file change is a core change that needs to be released with Airflow core not with providers. This also means that the connection will be available only to Airflow 2.7.0+
WDYT?

Reply:
@eladkal , Is the provider going to be released before 2.7.0? I think it shouldn't since it touched core because the connection entry is needed to use it in any Airflow version, right?.

Reply:
I created https://github.com/apache/airflow/issues/32048 to hopefully solve it once and for all

As for your question - the provider must support minimum version of Airflow 2.4.0